packages/client/src/client/App/Credit/CreditRfqs/CreditRfqCards/Quote/styled.ts

```diff
@@ -77,8 +77,8 @@ export const QuoteDot = styled.div<AnimatedRowProps>`
     highlight
       ? theme.textColor
       : passed
-      ? theme.primary[4]
-      : getBuySellHighlightTextColor(theme, direction)};
+        ? theme.primary[4]
```

Comment:
prettier 3 - indent nested ternaries (overdue, that one .. making me think we should never nest ternaries ....)